docs/user_guide/productionizing/configuring_access_to_gpus.md

```diff
@@ -6,31 +6,405 @@
+from flytekit.extras.accelerators import A100
+
+image = ImageSpec(
+    base_image= "ghcr.io/flyteorg/flytekit:py3.10-1.10.2",
+     name="pytorch",
+     python_version="3.10",
+     packages=["torch"],
+     builder="envd",
+     registry="<YOUR_CONTAINER_REGISTRY>",
+ )
+
+@task(requests=Resources( gpu="1"),
+              accelerator=A100, 
+              ) 
+def gpu_available() -> bool:
+   return torch.cuda.is_available()
+```
+
+#### How it works?
+
+By default, the task is scheduled on a `2g.10gb` MIG partition.
+
+`flytepropeller` only injects the node selector that matches nodes with an `A100` device:
+
+```yaml
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: nvidia.com/gpu.accelerator
+            operator: In
+            values:
+            - nvidia-tesla-a100
+```
+
+
+### Request an unpartitioned A100 device
+The goal is to run the task using the resources of the entire A100 GPU:
+
+```python
+from flytekit import ImageSpec, Resources, task
+from flytekit.extras.accelerators import A100
+
+image = ImageSpec(
+    base_image= "ghcr.io/flyteorg/flytekit:py3.10-1.10.2",
+     name="pytorch",
+     python_version="3.10",
+     packages=["torch"],
+     builder="envd",
+     registry="<YOUR_CONTAINER_REGISTRY>",
+ )
+
+@task(requests=Resources( gpu="1"),
+              accelerator=A100.unpartitioned, 
+              ) # request the entire A100 device
+def gpu_available() -> bool:
+   return torch.cuda.is_available()
+```
+
+#### How it works?
+
+When this task is evaluated `flytepropeller` injects a node selector expression that only matches nodes where the label specifying a partition size is **not** present:
+
+```yaml
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: nvidia.com/gpu.accelerator
+            operator: In
+            values:
+            - nvidia-tesla-a100
+          - key: nvidia.com/gpu.partition-size
+            operator: DoesNotExist
+```
+The expression can be controlled from the Helm values:
+
+
+**flyte-core**
+```yaml
+configmap:
+  k8s:
+    plugins:
+      k8s:
+        gpu-unpartitioned-node-selector-requirement :
+          key: cloud.google.com/gke-gpu-partition-size #change to match your node label configuration
+          operator: Equal
+          value: DoesNotExist
+```
+**flyte-binary**
+```yaml
+configuration:
+  inline:
+    plugins:
+      k8s:
+        gpu-unpartitioned-toleration:
+          gpu-unpartitioned-node-selector-requirement :
+          key: cloud.google.com/gke-gpu-partition-size #change to match your node label configuration
+          operator: Equal
+          value: DoesNotExist
+```
+
+
+Scheduling can be further controlled by setting in the Helm chart a toleration that `flytepropeller` injects into the task pods:
```

Comment:
```suggestion
Scheduling can be further controlled by setting a toleration in the Helm chart that `flytepropeller` injects into the task pods:
```

Reply:
Maybe it's because English is not my native tongue :) but I read the suggestion as if it's the Helm chart that is injected

Reply:
Oh, I see what you mean -- you can disregard my suggestion, then, the original text is much clearer!

Reply:
Thank you!